coin-flip/script.js

```diff
@@ -2,12 +2,13 @@ function getRandNumber(max) {
 	return Math.floor(Math.random() * max);
 }
 
-var number = getRandNumber(2);
 
-var headsOrTails = "None";
+const flipCoin = () => {
+	const number = getRandNumber(2);
 
-if (number == 1) {
-	document.getElementById("coin").innerHTML = "<h1>You flipped Heads!</h1>";
-} else {
-	document.getElementById("coin").innerHTML = "<h1>You flipped Tails!</h1>";
+	const headsOrTails = number === 1 ? "Heads" : "Tails";
+
+	const newHtml = `<h1>You flipped ${headsOrTails}!`;
```

Comment:
Using back-ticks allows for string interpolation or [template strings/literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in your code. That means you can use `${}` around some JS variable and insert it into your code.